core/src/toga/window.py

```diff
@@ -272,6 +272,9 @@ def close(self) -> None:
         undefined, except for :attr:`closed` which can be used to check if the window
         was closed.
         """
+        if getattr(self, "content", None) is not None:
```

Comment:
In what conditions can content *not* exist as an attribute? It's one of the first attributes set on Window

Reply:
sorry, I meant to do:
```
if self.content:
```